.pre-commit-config.yaml

```diff
@@ -25,8 +27,7 @@ repos:
     hooks:
       - id: black
         language_version: python3.10
-        verbose: true
-        args: ["--verbose"]
+        args: [--config=pyproject.toml]
```

Comment:
The `--config-pyproject.toml` arg might not be needed, but consistency seemed worthwhile.

Also - I'm okay bringing back `verbose` if we need it, but was very spammy on my machine when `--all-files` was passed as an arg.

(I also tried sending an explicit config path to `isort` - but this had an adverse affect so I reverted it.)